dev/setup-fleet-downstream

```diff
@@ -54,13 +54,16 @@ else
 fi
 
 # docker network inspect bridge -f '{{(index .IPAM.Config 0).Gateway}}'
-public_hostname="${public_hostname-172.17.0.1.omg.howdoi.website}"
+# public_hostname="${public_hostname-172.17.0.1.omg.howdoi.website}"
+
+# works due to same network of k3d clustres and patched SAN cert
+public_hostname="${public_hostname-k3d-upstream-server-0}"
 
 kubectl config use-context "$downstream_ctx"
 helm -n cattle-fleet-system upgrade --install --create-namespace --wait fleet-agent charts/fleet-agent \
   --set-string labels.env=test \
   --set apiServerCA="$ca" \
-  --set apiServerURL="https://$public_hostname:36443" \
```

Comment:
> --api-port 6550 is not required for the example to work.
> It’s used to have k3s‘s API-Server listening on port 6550 **with that port mapped to the host system.**

-- https://k3d.io/v5.0.0/usage/exposing_services/

As this maps the port to the host system and the idea is to not involve the host system for cluster registrations by going through the Kubernetes internal service, I don't see why it would cause an issue not to use it.